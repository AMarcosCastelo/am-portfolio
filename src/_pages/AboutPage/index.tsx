import * as S from './styles'

const AboutPage = () => (
  <S.Wrapper>
    <h1>AboutPage</h1>
  </S.Wrapper>
)

export default AboutPage
