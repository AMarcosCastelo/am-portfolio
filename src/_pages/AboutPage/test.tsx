import { render, screen } from '@testing-library/react'

import AboutPage from '.'

describe('<AboutPage />', () => {
  it('should render the heading', () => {
    const { container } = render(<AboutPage />)

    expect(
      screen.getByRole('heading', { name: /AboutPage/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
