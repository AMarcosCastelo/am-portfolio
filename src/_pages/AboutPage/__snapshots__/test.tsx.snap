// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    AboutPage
  </h1>
</main>
`;
